### PR TITLE
jsonrpc-apis,  ws-apis and ipc-apis update

### PR DESCRIPTION
**Pull request by Tbaut**

### CODE REVIEW

File: parity/cli/mod.rs [hunk 1]
```diff
@@ -452,7 +452,7 @@ usage! {
 
 			ARG arg_jsonrpc_apis: (String) = "web3,eth,pubsub,net,parity,parity_pubsub,traces,rpc,shh,shh_pubsub", or |c: &Config| c.rpc.as_ref()?.apis.as_ref().map(|vec| vec.join(",")),
 			"--jsonrpc-apis=[APIS]",
-			"Specify the APIs available through the JSONRPC interface. APIS is a comma-delimited list of API name. Possible name are all, safe, web3, eth, net, personal, parity, parity_set, traces, rpc, parity_accounts, pubsub, parity_pubsub, shh, shh_pubsub, signer, secretstore. You can also disable a specific API by putting '-' in the front: all,-personal.",
+			"Specify the APIs available through the JSONRPC interface. APIS is a comma-delimited list of API name. Possible names are: all, safe, web3, net, eth, pubsub, personal, signer, parity, parity_pubsub, parity_accounts, parity_set, traces, rpc, secretstore, shh, shh_pubsub. You can also disable a specific API by putting '-' in the front, example: all,-personal. safe contains following apis: traces, parity_pubsub",
```
Comment (tomusdrw) [inline parity/cli/mod.rs]: Actually `safe` contains all APIs from public list:
https://github.com/paritytech/parity/blob/2e09db185f4cd443a4753c79458ce1105f60fcea/parity/rpc_apis.rs#L623-L632

plus `traces` and `ParityPubSub`.

**Comment by Tbaut** [comment]
Thanks @tomusdrw for spotting this :)

**Comment by ddorgan** [comment]
[clabot: check]

**Comment by ddorgan** [comment]
[clabot:check]

File: parity/cli/mod.rs [hunk 2]
```diff
@@ -452,7 +452,7 @@ usage! {
 
 			ARG arg_jsonrpc_apis: (String) = "web3,eth,pubsub,net,parity,parity_pubsub,traces,rpc,shh,shh_pubsub", or |c: &Config| c.rpc.as_ref()?.apis.as_ref().map(|vec| vec.join(",")),
 			"--jsonrpc-apis=[APIS]",
-			"Specify the APIs available through the JSONRPC interface. APIS is a comma-delimited list of API name. Possible name are all, safe, web3, eth, net, personal, parity, parity_set, traces, rpc, parity_accounts, pubsub, parity_pubsub, shh, shh_pubsub, signer, secretstore. You can also disable a specific API by putting '-' in the front: all,-personal.",
+			"Specify the APIs available through the JSONRPC interface. APIS is a comma-delimited list of API name. Possible names are: all, safe, web3, net, eth, pubsub, personal, signer, parity, parity_pubsub, parity_accounts, parity_set, traces, rpc, secretstore, shh, shh_pubsub. You can also disable a specific API by putting '-' in the front, example: all,-personal. safe contains following apis: web3, net, eth, pubsub, parity, parity_pubsub, traces, rpc, shh, shh_pubsub",
```
Comment (niklasad1) [inline parity/cli/mod.rs]: s/APIS is a comma-delimited list of API name/The APIs is a comma-delimted list of API names.

Comment (niklasad1) [reply parity/cli/mod.rs]: If you agree change at the other places as well otherwise ignore it :)

Comment (Tbaut) [reply parity/cli/mod.rs]: Agree

**Pull request merged**